operator/src/main/java/oracle/kubernetes/operator/MakeRightDomainOperation.java

```diff
@@ -36,14 +36,6 @@ public interface MakeRightDomainOperation extends PacketComponent {
    */
   MakeRightDomainOperation withEventData(EventData eventData);
 
-  /**
-   * Specifies that the current MakeRightOperation may skip updating the DomainStatus if the
-   * DomainPresenceInfo.ServerStartupInfo value has not yet been constructed..
-   */
-  default MakeRightDomainOperation skipUpdateDomainStatusIfNeeded() {
```

Comment:
Why is this removed? I thought that this was part of Lenny's fix for the intermittent incorrect status from the status thread when the make-right and status updater fibers ran at the same time...

Reply:
Actually, I thought we did remove it as Anil and I had changed it as it wasn't needed anymore as we used the MakeRightDomainOperation.wasStartedFromEvent() method instead.

Reply:
Sounds good.

Reply:
@lennyphan can you comment?

Reply:
Already commented above, it should be removed as it's not used anymore.  We mistakenly left it when it should have been cleaned up.  This issue is resolved.